lib/services.js

```diff
@@ -168,6 +169,14 @@ const services = {
         // will reassign some of the above values with the ones
         // from the intiation of the multipart upload
         // (for instance, storage class)
+        const validateUserMdRes = validateUserMetadata(metaHeaders);
```

Comment:
I moved it from services storing md step to when the metaheaders object is defined for each of the apis.